export_saved.py

```diff
@@ -19,9 +19,10 @@
 class Converter():
     """Converts a CSV instapaper export to a Chrome bookmark file."""
 
-    def __init__(self, file):
+    def __init__(self, file, html_file):
```

Comment:
Why not directly?
~~~ python
def __init__(self, file, html_file='chrome-bookmarks.html'):
~~~